### ISSUE TITLE
tink template create should also accept STDIN

### ISSUE THREAD

**Issue by alexellis** [tinkerbell/tink#121]
tink template create should also accept STDIN to match the change made to the workflow command

Rather than `tink template create --name ubuntu_provisioning --path /tmp/tmp`

`cat /tmp/tmp | tink template create --name ubuntu_provisioning`

or 

`cat | tink template create --name ubuntu_provisioning` followed by control+v

This is especially useful when having to run the tink CLI in a container, since there's no static build yet as per issue #73

**Comment by alexellis** [comment]
@gauravgahlot @parauliya PTAL?

**Comment by gauravgahlot** [comment]
@alexellis Yeah, I agree. All commands expecting files should support both: a flag and `stdin`.

**Comment by rgl** [comment]
As a workaround, in Linux/bash at least, you can use `/dev/stdin`. e.g.:

```bash
docker exec -i deploy_tink-cli_1 tink template create --name hello-world --path /dev/stdin <<EOF
version: '0.1'
global_timeout: 600
tasks:
  - name: hello world
    worker: {{.device_1}}
    actions:
      - name: hello-world
        image: hello-world
        timeout: 60
EOF
```
